src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/AdjournCaseIt.java

```diff
@@ -1,21 +1,20 @@
 package uk.gov.hmcts.reform.sscs.callback;
```

Comment:
This file looks like it has largely been copied from 10950? Which parts are unique to your branch? Perhaps changing the PR to target 10950 rather than master would make sense?